File: Python/Clase-1/Ejercicios/Ejercicio-7.py
# Dada la siguiente tupla
tupla = (13, 1, 8, 3, 2, 5, 8)  # Definimos la tupla

lista = []#definimos una lista vacia
#Filtrar los elementos de la tupla que sean menores a 5
for elemento in tupla:
    if elemento < 5:
        lista.append(elemento)
print(lista)

File: JavaScript/Clase-1/Ejercicios/Ejercicio-7.js
const tupla = [13, 1, 8, 3, 2, 5, 8]; // Definimos el array (tupla en Python)
const lista = [];

for (let elemento of tupla) {
    if (elemento < 5) {
        lista.push(elemento);
    }
}

console.log(lista);
